src/components/Doc/ComponentDoc.js

```diff
@@ -49,6 +50,21 @@ export const ComponentDoc = ({
         <title>{pageTitle || name}</title>
         <meta name="description" content={description} />
       </Helmet>
+      {stable && (
+        <Notification
+          background="light-3"
+          status="info"
+          global
+          actions={[
+            {
+              label: 'stable branch.',
```

Comment:
I prefer to limit the anchor to the actual subject being linked to, and have the punctuation live outside. Soft opinion.

Out of curiosity, is there an advantage to using the `actions` prop for the link vs. inline with the message? Mmmmm.... seeing that `message` only accepts a string currently -- supporting a node may be a future enhancement. So, this is okay for now.

Reply:
I believe when we added the `actions` and `messages` we wanted to limit users and how they can pass something to `messages` but in grommet world we should expand this to support a node

Reply:
Yes I agree on having the punctuation outside of the anchor. The way 
Notification is built the actions is always placed at the end so I wasn't able to put the punctuation after the anchor. As you noted `message` only accepts a string I can open an issue on grommet for an enhancement that would let us pass a node.

Reply:
Issue created: https://github.com/grommet/grommet/issues/6320